src/applications/edu-benefits/10203/containers/IntroductionPage.jsx

```diff
@@ -302,4 +270,4 @@ const mapDispatchToProps = {
 export default connect(
   mapStateToProps,
   mapDispatchToProps,
-)(withFeatureToggle(IntroductionPage));
+)(IntroductionPage);
```

Comment:
no longer have to wrap with a HOC because we've removed the need for a feature toggle check